src/core/server/saved_objects/migrationsv2/actions/es_errors.test.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { isIncompatibleMappingException, isWriteBlockException } from './es_errors';
+
+describe('isWriteBlockError', () => {
+  it('returns true for a `index write` cluster_block_exception', () => {
+    expect(
+      isWriteBlockException({
```

Comment:
Maybe we can add an integration test instead? Since it's easy to reproduce for an ES instance.

Reply:
There is already an integration test for the action using the function, but it doesn't hurt to do it for the helper itself.